app/models/tree-component.ts

```diff
@@ -0,0 +1,22 @@
+import { IActionMapping } from "angular-tree-component/dist/angular-tree-component";
+
+export enum NodeState {
```

Comment:
This is going to conflict with the NodeState if you have it in the models folder